src/Components/Forms/src/Microsoft.AspNetCore.Components.Forms.csproj

```diff
@@ -11,6 +11,7 @@
 
   <ItemGroup>
     <Reference Include="Microsoft.AspNetCore.Components" />
+    <Reference Include="Microsoft.AspNetCore.Http.Abstractions" />
```

Comment:
This is problematic.

We can't depend on stuff that brings in `HttpContext` and a bunch of other framework related types. Is there anything in the validation stuff that requires things from `Microsoft.AspNetCore.Http`?

This is more of a question for @captainsafia

Reply:
Per our offline discussion, let's create a new `Microsoft.Extensions.Validation` package under `src/Validation` and move things into it there.

Happy to help if you need any guidance moving the testing infrastructure...

We'll also want to consider whether the source generator goes into the extensions package or not. My inclination is to say it does. In that case, we need to add more sanity checks to it to handle cases where the package might be referenced outside the context of ASP.NET Core. Specifically, this would just be better fallbacks for places where it looks for symbols that are defined in ASP.NET Core namespaces...